packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -8,134 +12,223 @@ import type { AssetsContractController } from './AssetsContractController';
 import type { Token } from './TokenRatesController';
 import type { TokensState } from './TokensController';
 
-// TODO: Remove this export in the next major release
-export { BN };
+const DEFAULT_INTERVAL = 180000;
+
+const controllerName = 'TokenBalancesController';
+
+const metadata = {
+  contractBalances: { persist: true, anonymous: false },
+};
 
 /**
- * @type TokenBalancesConfig
+ * @type TokenBalancesControllerOptions
  *
```

Comment:
It isn't necessary to use `@type` when documenting types, and this is the wrong tag, anyway (JSDoc recognizes `@typedef` as a way of documenting types, but TypeDoc ignores this tag). So we can remove this:

``` suggestion
```